src/NotepadNext/SessionManager.h

```diff
@@ -0,0 +1,41 @@
+/*
+ * This file is part of Notepad Next.
+ * Copyright 2022 Justin Dailey
+ *
+ * Notepad Next is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Notepad Next is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Notepad Next.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+
+#ifndef SESSIONMANAGER_H
```

Comment:
You should start using `#pragma once` it does the same, and is supported on GCC since forever. Do you want me to handle this in another PR?

Reply:
> is supported on GCC since forever

Interesting. Guess I always thought of `#pragma once` as a MSVC++ specific feature (I first found it many years ago!), but as you mentioned it appears it has been universally supported for a while now.

So yes I agree `#pragma once` is desirable. You are certainly welcome to submit the changes. Only need to worry about code under `.src/NotepadNext`